Calibration/HcalCalibAlgos/macros/CalibCorr.C

```diff
@@ -0,0 +1,173 @@
+#include <algorithm>
+#include <map>
+#include <vector>
+#include <string>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+class CalibCorr {
+public :
+  CalibCorr(const std::string& infile, bool debug=false);
+  ~CalibCorr() {}
+
+  float getCorr(int run, unsigned int id);
+private:
+  void                     readCorr(const std::string& infile);
+  std::vector<std::string> splitString(const std::string&);
+  unsigned int getDetIdHE(int ieta, int iphi, int depth);
+  unsigned int getDetId(int subdet, int ieta, int iphi, int depth);
+  unsigned int correctDetId(unsigned int detId);
+
+  static const unsigned int nmax_=10;
```

Comment:
please make this constexpr (although it likely doesn't matter much in a ROOT macro)

Reply:
Did you try constexpr in ROOT macro with cint - it does not work for 5.45

Reply:
It does not work for 5.26

Reply:
ok, if the macro has to cover several root versions, it's fine, but why do you use such an old version?
the default one is ROOT 6 in 94X